test-framework/test-suites/unit/files/pallet_artifacts/SLE-15-SP2-Full-x86_64-PublicBeta-Media2.iso.media

```diff
@@ -0,0 +1,3 @@
+SUSE - SLE-15-SP2-Full-x86_64-Build136.2-Media-SOURCE
+SLE-15-SP2-Full-x86_64-Build136.2
+1
```

Comment:
Missing trailing newline. **twitch**

check throughout.

Reply:
Why are trailing newlines relevant? This isn't c code! 

In particular, this file should not have any characters added, as it should be an exact copy from the sles media.

Reply:
git has always complained about them, so I thought I should listen to father git. If this is supposed to be a copy of a file from a SLES media, then it should be the exact same.

Reply:
I feel led astray @ryancausey 

Removed those newlines, against the recommendations of father GH.